testdata/p4_16_samples_outputs/array_field.p4-stderr

```diff
@@ -0,0 +1,12 @@
+array_field.p4(27): [--Wwarn=invalid_header] warning: accessing a field of an invalid header s[a]
+        s[a].z = 1;
+        ^^^^
+array_field.p4(28): [--Wwarn=invalid_header] warning: accessing a field of an invalid header s[+]
+        s[a+1].z = 0;
+        ^^^^^^
+array_field.p4(29): [--Wwarn=invalid_header] warning: accessing a field of an invalid header s[tmp]
+        a = f(s[a].z, 0);
+              ^^^^
+array_field.p4(30): [--Wwarn=invalid_header] warning: accessing a field of an invalid header s[tmp_0]
+        a = f(s[a].z, 1);
+              ^^^^
```

Comment:
Also, I've noticed a minor bug here. This warning says that it is accessed to a field `s[tmp_0]` instead of `s[a]`, but the underlined expression is correct.

Reply:
I am not sure this can be easily fixed.

Reply:
You should probably file an issue about it as soon as you can (probably after we merge this).